packages/react/src/themes/teams/components/Toolbar/toolbarCustomItemStyles.ts

```diff
@@ -0,0 +1,47 @@
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+import { ToolbarCustomItemProps } from '../../../../components/Toolbar/ToolbarCustomItem'
+import { ToolbarVariables } from './toolbarVariables'
+import { getColorScheme } from '../../colors'
+
+const toolbarCustomItemStyles: ComponentSlotStylesInput<
+  ToolbarCustomItemProps,
+  ToolbarVariables
+> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    const colors = getColorScheme(v.colorScheme)
+
+    return {
+      backgroundColor: v.background,
+      borderColor: 'transparent',
+      borderWidth: v.borderWidth,
+      borderStyle: 'solid',
+      height: v.itemHeight,
+      color: v.foreground || colors.foreground1,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      ...(p.fitted !== true &&
```

Comment:
just to be sure - which case we are protecting against with explicit `p.fitted !== true && ..`, instead of `!p.fitted && ..`?

Reply:
```jsx
const hasPadding = (fitted) => fitted !== true && fitted !== 'horizontally'
const hasPaddingModified = (fitted) => !fitted && fitted !== 'horizontally'
```

```js
console.log(hasPadding(true), hasPadding('horizontally'), hasPadding('vertically'))
```

```
false false true
```

```jsx
console.log(hasPaddingModified(true), hasPaddingModified('horizontally'), hasPaddingModified('vertically'))
```
```
false false false
```

I believe it's the reason